README.md

```diff
@@ -131,7 +131,7 @@ git clone https://github.com/OSGeo/gdal.git
 cd gdal/gdal
 git checkout tags/2.1.0
 ./configure
-make install
+sudo make install
```

Comment:
I never needed sudo 🤔 

Maybe you used sudo at an earlier stage and messed up the dir permissions? This is why I avoid sudo at all cost :/